### ISSUE TITLE
Can't install kontena-server due to wrong docker dependency

### ISSUE THREAD

**Issue by industral** [kontena/kontena#1947]
```
$ sudo apt-get install kontena-server
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 kontena-server : Depends: docker-engine (>= 1.9) but it is not going to be installed or
                           docker.io (>= 1.11) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

Docker now install as `docker-ce` or `docker-ee`

```
$ docker -v
Docker version 17.03.0-ce, build 3a232c8
```

Checkout http://docker.com or https://store.docker.com/editions/community/docker-ce-server-ubuntu

**Comment by SpComb** [comment]
We'll put out a 1.1.4 release to fix this + the missing bintray Ubuntu packages for 1.1.3.

Meanwhile, you should still be able to install using the Ubuntu `xenial-updates` `docker.io` package?

**Comment by SpComb** [comment]
Version 1.1.4 is now available on bintray: https://bintray.com/kontena/ubuntu

```
     1.1.4-1 500
        500 http://dl.bintray.com/kontena/ubuntu xenial/main amd64 Packages
     1.1.4-1 500
        500 http://dl.bintray.com/kontena/ubuntu xenial/main amd64 Packages
```

The [1.1.4 release](https://github.com/kontena/kontena/releases/tag/v1.1.4) contains only Ubuntu packaging fixes, and is otherwise identical to the [1.1.3 release](https://github.com/kontena/kontena/releases/tag/v1.1.3), which is not available for Ubuntu. Upgrade any Ubuntu installs directly from 1.1.2 -> 1.1.4 for the fixes in 1.1.3.

The 1.1.4 Ubuntu packages will work with the new `docker-ce` `17.03.*` packages.
